vector/src/main/res/values/theme_status.xml

```diff
@@ -8,12 +8,18 @@
         <!-- default background color -->
         <item name="riot_primary_background_color">@color/riot_primary_background_color_status</item>
 
+        <!-- waiting view background -->
+        <item name="waiting_background_color">#AAAAAAAA</item>
```

Comment:
What about other elements defined in style `Theme.Vector.Light` and still missing here?
Actually the comment `<!-- Only setting the items we need to override, otherwise inheriting -->` comes from the `Black` theme, which inherit the `Dark` theme (and not directly an `AppCompat` theme). I think you can remove this comment and add the missing elements (copy them from `Theme.Vector.Light` is ok for a first step.